pkg/sentry/platform/ring0/defs_amd64.go

```diff
@@ -76,15 +76,42 @@ type KernelOpts struct {
 type KernelArchState struct {
 	KernelOpts
 
+	// cpuEntries is array of kernelEntry for all cpus
+	cpuEntries []kernelEntry
+
 	// globalIDT is our set of interrupt gates.
-	globalIDT idt64
+	globalIDT *idt64
```

Comment:
Optional: add a comment that explains this is a pointer because the idt will be exposed in the upper half, and the rest of the bits will be exposed on a per-cpu basis?